src/test/java/no/unit/nva/model/contexttypes/BookTest.java

```diff
@@ -0,0 +1,64 @@
+package no.unit.nva.model.contexttypes;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.Level;
+import nva.commons.utils.IoUtils;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class BookTest {
```

Comment:
I am wondering whether it would be a good idea to have two methods `fromJson(String json)` (static) and `toJson()` that would also include  serialization error handling. But this maybe should be a different PR.